template/lambda-sqs-worker-cdk/infra/index.ts

```diff
@@ -8,4 +8,10 @@ const app = new App();
 // eslint-disable-next-line no-new
 new AppStack(app, 'appStack', {
   stackName: config.appName,
+  tags: {
+    'seek:env:label': config.workerLambda.environment.ENVIRONMENT,
```

Comment:
This is kinda gross...

Reply:
Could we just store environment at the top-level? or 

```
export const environment = Env.oneOf(ENVIRONMENTS)('ENVIRONMENT');
```